app/src/main/java/com/gatheringhallstudios/mhworlddatabase/adapters/SkillTreeAdapterDelegate.kt

```diff
@@ -20,10 +20,12 @@ class SkillTreeAdapterDelegate(private val onSelected: (SkillTreeView) -> Unit)
     override fun bindView(view: View, data: SkillTreeView) {
         val icon = view.context.getVectorDrawable(R.drawable.ic_armor_skill, data.icon_color)
 
-        with (view as IconLabelTextCell) {
+        with(view as IconLabelTextCell) {
             view.setLeftIconDrawable(icon)
             view.setLabelText(data.name)
             view.removeDecorator()
         }
+
+        view.setOnClickListener { (onSelected(data)) }
```

Comment:
not a problem for merging, but the parenthesis around onSelected are superfluous